From 9d1c39b2115d3e7a403715083a584818f10feba2 Mon Sep 17 00:00:00 2001
From: Peter Hutterer <peter.hutterer@who-t.net>
Date: Thu, 21 Jun 2012 14:26:15 +1000
Subject: [PATCH] os: print newline after printing display name

Much easier for scripts that try to read the display value off the file
descriptor. Plus, this restores the behaviour we had for this patch in
Fedora since server 1.6 (April 2009).

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
---
 os/connection.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/os/connection.c b/os/connection.c
index 039942f..5144d16 100644
--- a/os/connection.c
+++ b/os/connection.c
@@ -423,7 +423,7 @@ CreateWellKnownSockets(void)
         }
         if (!found)
             FatalError("Failed to find a socket to listen on");
-        snprintf(dynamic_display, sizeof(dynamic_display), "%d", i);
+        snprintf(dynamic_display, sizeof(dynamic_display), "%d\n", i);
         display = dynamic_display;
     }
 
-- 
1.7.10.2

